specifyweb/frontend/js_src/lib/components/Reports/index.tsx

```diff
@@ -216,7 +217,9 @@ function ReportRow({
               />
             </Button.LikeLink>
           </th>
-          <th>{getField(tables.SpReport, 'createdByAgent').label}</th>
+          <th className="link pointer-events-none">
```

Comment:
pointer-events-none sounds good.

as per https://github.com/specify/specify7/pull/4396#discussion_r1470463857, could you also do this refactor?

```suggestion
          <th className={`${className.link} pointer-events-none`}>
```